recipes/blast/build.sh

```diff
@@ -42,11 +39,8 @@ LIB_INSTALL_DIR=$PREFIX/lib/ncbi-blast+
 # -vdb: disable VDB/SRA toolkit
 # z: set zlib
 # bz2: set libbz2
-# boost: set boost
```

Comment:
I take it blast no longer uses boost?

Reply:
No, just the unit tests do (which were being deleted at the end of the recipe)